locales/it/index.yml

```diff
@@ -1,5 +1,7 @@
 ---
 global:
+  alert:
+    call: "Vuoi chiamare #?"
```

Comment:
There is a formal way to define variable inside a yml value
https://github.com/teamdigitale/io-app/blob/7c3d5e999070641893eafbb121e396aa3cbe74e5/locales/en/index.yml#L591

Btw could you add another line to report the number?
ex: 
```
Do you want to call SERVICE_NAME?
+39 1234 567
```

Reply:
No longer necessary